app/uk/gov/nationalarchives/omega/editorial/services/EditSetService.scala

```diff
@@ -21,20 +21,22 @@
 
 package uk.gov.nationalarchives.omega.editorial.services
 
+import cats.effect.unsafe.implicits.global
 import play.api.Logger
-import uk.gov.nationalarchives.omega.editorial.editSets
+import uk.gov.nationalarchives.omega.editorial.connectors.ApiConnector
 import uk.gov.nationalarchives.omega.editorial.models.EditSet
 
-import javax.inject.Singleton
+import javax.inject.{ Inject, Singleton }
+import scala.concurrent.Future
 
 @Singleton
-class EditSetService {
+class EditSetService @Inject() (apiConnector: ApiConnector) {
 
   private val logger: Logger = Logger(this.getClass)
 
-  def getEditSet(id: String): EditSet = {
+  def getEditSet(id: String): Future[EditSet] = {
```

Comment:
Given the context, could this merely name this `get`?

Reply:
I think that makes sense, yes.

Reply:
Confirmed.